frontend/src/modules/application/components/ApplicationForm.jsx

```diff
@@ -14,64 +22,65 @@ const TextboxInput = ({ label, curValue, onChange, required }) => <div>
               required={required}/>
 </div>
 
-const BinaryRadioInput = ({ label, curValue,required }) => <div>
+// Binary radio input field component
+const BinaryRadioInput = ({ label, curValue, onChange, required }) => <div>
     <label style={{ width: "20vw" }}>{required ? label + ':' : label}</label>
     <label>
-        <input type="radio" checked={curValue === "true"}/>
+        <input type="radio" value="true" checked={curValue === "true"} onChange={onChange}/>
         Yes
     </label>
     <label>
-        <input type="radio" checked={curValue === "false"}/>
+        <input type="radio" value="false" checked={curValue === "false"} onChange={onChange}/>
         No
     </label>
 </div>
 
-
-
-const allFields = [].concat.apply([], 
-    [personalInformationFields, 
-    currentProgramInformationFields,
-    currentStatusFields, 
-    customQuestions])
-
-const initialState = allFields.reduce((acc, cur) => ({ ...acc, [cur.value]: "" }), {})
-
+// Application form component that holds multiple individual simple components defined earlier
```

Comment:
Again, perhaps this is something we should debate given the nature of dev's joining this project, who may not be familiar with react. I feel like this comment is trying to indicate the nature of the framework, in other words, that the code already speaks sufficiently for itself. Not sure it's really necessary to keep.